ownCloudSDK/Core/Sharing/OCCore+Sharing.m

```diff
@@ -383,12 +384,12 @@ - (nullable NSProgress *)makeDecisionOnShare:(OCShare *)share accept:(BOOL)accep
 				// - files, folders
 				[[OCShareRole alloc] initWithType:OCShareRoleTypeViewer
 						       shareTypes:OCShareTypesMaskUserShare|OCShareTypesMaskGroupShare
-						     permissions:OCSharePermissionsMaskRead|OCSharePermissionsMaskShare
+						     permissions:OCSharePermissionsMaskRead|(resharingSupported ? OCSharePermissionsMaskShare : 0)
 					 customizablePermissions:OCSharePermissionsMaskNone
 						       locations:OCLocationTypeFile|OCLocationTypeFolder
 						      symbolName:@"eye.fill"
 						   localizedName:OCLocalized(@"Viewer")
-					    localizedDescription:OCLocalized(@"Download, preview and share")],
+					    localizedDescription:OCLocalized(resharingSupported ? @"Download, preview and share" : @"Download and preview")],
 
```

Comment:
`OCLocalized` needs to be changed to `OCLocalizedString` to get in sync with the new translation changes before merging.

Reply:
Fixed.